flax/nn/utils.py

```diff
@@ -95,3 +95,66 @@ def _level_of_value(xs):
       level = _trace_level(x._trace.master)
       max_level = max(level, max_level)
   return max_level
+
+
+def model_summary(model: nn.Model):
```

Comment:
Can wee use something more generic than nn.Model? dict o list with the parameteres?